src/govuk/components/footer/template.test.js

```diff
@@ -207,4 +207,36 @@ describe('footer', () => {
       expect($sectionBreak.length).toBeFalsy()
     })
   })
+
+  describe('content licence', () => {
+    it('is visible', () => {
+      const $ = render('footer', examples.default)
+
+      const $licenceMessage = $('.govuk-footer__licence-description')
+      expect($licenceMessage.text().trim()).toContain('Open Government Licence v3.0')
+    })
+
+    it('can be customised', () => {
```

Comment:
Similar to the examples comment above, might be good to have tests for both `text` and `html`. We've also previously written tests to check that HTML passed into the `text` option renders as text.